GHAS-on-GHES-feature-matrix.md

```diff
@@ -5,14 +5,18 @@ This document helps answer the question "is this GHAS feature available in my ve
 The following tables include notable feature releases for GitHub Advanced Security.  Each row represents a feature.  The columns in the row indicate the level of support for each **supported** Enterprise Server release.  Are your repositories hosted on github.com? All of these features are already available for you :+1:.
 
 #### Contents
-- [Secret scanning](#secret-scanning)
-- [Code scanning](#code-scanning)
-- [Supply-chain security](#supply-chain-security)
-  - [Dependabot alerts](#dependabot-alerts)
-  - [Dependabot security updates](#dependabot-updates)
-  - [Dependency review and submission api](#dependency-review-and-submission-api) 
-- [Security Overview](#security-overview)
-- [Administration](#administration)
+- [GitHub Advanced Security (GHAS) Feature Matrix](#github-advanced-security-ghas-feature-matrix)
```

Comment:
Very nice, are you using a tool to edit markdown / TOC / tables for this doc 👀 !

Reply:
No, I used Codespaces, with the editor and the renderer. I thought about looking for a tool, but it would have taken the more time to find the perfect tool than making that change. 😄